doc/source/data/doc_code/accessing_datasets.py

```diff
@@ -173,24 +172,21 @@ def consume(data: ray.data.Dataset[int]) -> int:
 
 # Specify the label column; all other columns will be treated as feature columns and
 # will be concatenated into the same TensorFlow tensor.
```

Comment:
This comment is no longer true.

Reply:
Also we may actually want to use concatenation for this example, so that it is closer to what consumers (e.g. Keras model) expect.

Reply:
Will remove this comment.